pawnShop/src/main/java/com/example/pawnShop/Dto/Product/ProductTypeDto.java

```diff
@@ -0,0 +1,17 @@
+package com.example.pawnShop.Dto.Product;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class ProductTypeDto {
```

Comment:
nit: Toshe i'm not quite sure you need to extend the Product entity ... you can use directly the Product.java class instead.
using it this way makes it more complicated from my point of view, instead i would've used directly an instance of the Product class and manipulated the records in the DB directly through it. You have to show me what you meant with this DTO, but yeah, instead i would delete this and directly use Product type everywhere. 
{I might be wrong, as i don't quite get the logic behind this extension, you have to explain me  }

Reply:
I think it's a good practice to use DTO's instead if expose the whole table

Reply:
Ok, I'm not familiar with this scenario overall, but i do believe you both, so keep it as it is 🤜🏻🤛🏻